fis/ut/fis_ut.c

```diff
@@ -47,6 +46,7 @@ static struct m0_rpc_client_ctx cctx = {
 static void fis_ut_motr_start(struct m0_rpc_server_ctx *rctx)
 {
 	int rc;
+	struct m0_net_xprt *m0_conf_ut_xprt = m0_net_xprt_get();
```

Comment:
Is the line 27 still needed?
If yes, at least the comment should be changed.

Reply:
Can we rename the `m0_conf_ut_xprt ` to `xprt`?
And accordingly the code below.

Reply:
conf/ut/common.h" is required for CLIENT_ENDPOINT_ADDR, SERVER_ENDPOINT_ADDR